File: Eraser2/app/src/main/java/com/luckly/eraser/ui/activity/splash/SplashActivity.java
package com.luckly.eraser.ui.activity.splash;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.os.Handler;
import android.view.WindowManager;

import com.luckly.eraser.R;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                finish();
                overridePendingTransition(R.anim.visible_effects, R.anim.invisible_effects);
            }
        }, 2500);
    }
}

File: Eraser2/app/src/main/java/com/luckly/eraser/ui/activity/splash/SplashActivity.kt
package com.luckly.eraser.ui.activity.splash

import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import com.luckly.eraser.R

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        Handler().postDelayed({
            finish()
            overridePendingTransition(R.anim.visible_effects, R.anim.invisible_effects)
        }, 2500)
    }
}